### ISSUE TITLE
SSH-like port tunnels

### ISSUE THREAD

**Issue by SparkyTD** [quasar/Quasar#1071]
**Is your feature request related to a problem? Please describe.**
It would be very helpful if Quasar supported TCP port tunnels similar to the ones offered by SSH. This way C2 could connect to any service on the target that's only locally available or otherwise blocked by the firewall, e.g. a database or an RDP server.

**Describe the solution you'd like**
The implementation would be similar to the way SOCKS5 proxies are implemented, with a list GUI that allows the user to create, edit or delete any number of remote or local tunnels.

**Comment by MaxXor** [comment]
Duplicate of #598

Indeed, would be a great addition.
